substrate/primitives/runtime-interface/src/impls.rs

```diff
@@ -57,37 +45,54 @@ macro_rules! impl_traits_for_primitives {
 			/// The type is passed directly.
 			impl RIType for $rty {
 				type FFIType = $fty;
+				type Inner = Self;
+			}
+
+			impl<'a> RIType for &'a $rty {
```

Comment:
Is it intentional that this is still around? I think you said that it should be removed.

Reply:
Yep. Removed it!